lib/indexers/projects/index.js

```diff
@@ -39,6 +52,7 @@ const indexProject = (esClient, project, ProjectVersion) => {
               licenceNumber: project.licenceNumber ? project.licenceNumber.toUpperCase() : null,
               licenceHolder: pick(project.licenceHolder, 'id', 'firstName', 'lastName'),
               establishment: pick(project.establishment, 'id', 'name'),
+              endDate: getEndDate(project),
```

Comment:
If a transferred project can't get revoked, then you might be able to just do this
```suggestion
              endDate: project.transferredOutDate || project.revocationDate || project.expiryDate,
```